tests/basic/storage_pool_test.go

```diff
@@ -4495,7 +4495,7 @@ var _ = Describe("{PoolExpandPendingUntilVolClean}", func() {
 			log.FailOnError(err, "Failed to check if Journal enabled")
 
 			log.InfoD("Current Size of the pool %s is %d", poolToBeResized.Uuid, poolToBeResized.TotalSize/units.GiB)
-			err = Inst().V.ExpandPool(poolToBeResized.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize, false)
+			err = Inst().V.ExpandPool(poolToBeResized.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize, true)
```

Comment:
please add a failure check wit false and then force it. so that the failure scenario is also verified

Reply:
reverted the fix for now , will update back once I hit the same issue again